source/community/release_notes/olive.rst

```diff
@@ -13,7 +13,7 @@ These are the release notes for the Olive release, the 15th community release of
  :depth: 1
  :local:
 
-Breaking Changes
+Breaking Bad Changes
 ================
```

Comment:
This line needs to be the same length as the words above

See https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html for how to make changes to your pull request!